Libraries/LibJS/Tests/builtins/functions/parseFloat.js

```diff
@@ -14,6 +24,16 @@ test("parsing numbers", () => {
     });
 });
 
+test("parsing zero", () => {
+    expect(isZeroNegative(parseFloat("0"))).toBe(false);
+    expect(isZeroNegative(parseFloat("+0"))).toBe(false);
+    expect(isZeroNegative(parseFloat("-0"))).toBe(true);
+
+    expect(isZeroNegative(parseFloat(0))).toBe(false);
+    expect(isZeroNegative(parseFloat(+0))).toBe(false);
+    expect(isZeroNegative(parseFloat(-0))).toBe(false);
```

Comment:
For all of these, we have `toBeTrue()` and `toBeFalse()` we should use here.

Reply:
Got it, changed 🫡